components/coachmark/stories/coachmark.mdx

```diff
@@ -0,0 +1,20 @@
+import { ArgTypes, Canvas, Meta, Description, Title } from '@storybook/blocks';
+
+import * as CoachmarkStories from './coachmark.stories';
+
+<Meta of={CoachmarkStories} title="Docs" />
+
+<Title of={CoachmarkStories} />
+<Description of={CoachmarkStories} />
+
+## Standard
+
+<Canvas of={CoachmarkStories.Default} />
```

Comment:
Also not necessarily a problem for this PR since it existed before but are you also seeing the top edge and left drop shadow being cut off like this? 🙃 
![image](https://github.com/adobe/spectrum-css/assets/54716846/9243de1d-e820-4fbb-b0f1-09248eebe648)

Reply:
It looks like this is [happening on prod](https://opensource.adobe.com/spectrum-css/preview/?path=/docs/components-coach-mark--docs) as well 🤔 But only on the docs page. It seems like the component butts right up to the container, so I went ahead and added some small top margin to the component in the template to fix this.